### PR TITLE
Update Metricbeat examples in docs

### PR DESCRIPTION
**Pull request by monicasarbu**
Update examples from the system module of Metricbeat.

cc-ed @andrewkroh @ruflin @dedemorton

### CODE REVIEW

File: metricbeat/docs/modules/system/core.asciidoc [hunk 1]
```diff
@@ -18,28 +18,30 @@ Below is an example document generated by this metricset.
...
-        "steal":0,
-        "system":7499682,
-        "system_p":0,
-        "user":9354158,
-        "user_p":0
-    },
-    "type":"metricsets"
+  "@timestamp": "2016-05-25T20:49:40.557Z",
+  "beat": {
+    "hostname": "mar.local",
```
Comment (andrewkroh) [inline metricbeat/docs/modules/system/core.asciidoc]: I propose we use the same host in each of these examples. How about `host.example.com`?

**Comment by andrewkroh** [comment]
Thanks for updating these. LGTM

**Comment by monicasarbu** [comment]
@andrewkroh I replaced all the hostnames with `host.example.com`.

**Comment by ruflin** [comment]
LGTM. Thanks for the updates. I'm trying to automate this in the future. Here is a first example that works: https://github.com/elastic/beats/blob/master/metricbeat/module/system/cpu/cpu_test.go#L12

Here is the code that does it and what test commands have to be used: https://github.com/elastic/beats/blob/master/metricbeat/mb/testing/data_generator.go#L15

In a later PR here we should also update the default host: https://github.com/elastic/beats/blob/master/metricbeat/mb/testing/data_generator.go#L38

**Pull request merged**